cumulusci/tasks/bulkdata/extract.py

```diff
@@ -251,8 +249,18 @@ def strip_name_field(record):
 
         self.session.commit()
 
-        if lookup_keys and not mapping["oid_as_pk"]:
-            self._convert_lookups_to_id(mapping, lookup_keys)
+    def _map_autopks(self):
+        # Convert Salesforce Ids to autopks
+        for m in self.mapping.values():
+            lookup_keys = list(m.get("lookups", {}).keys())
```

Comment:
You should be able to assume that the mapping has a `lookups` key and that it is a dict, thanks to https://github.com/SFDO-Tooling/CumulusCI/blob/master/cumulusci/tasks/bulkdata/mapping_parser.py#L71

Reply:
We should, but: `after` step mappings are synthesized in code and are still `dict`s. It is on my list to fix that and then go through and remove all `dict`-ish code throughout bulk data.